packages/manager/src/views/HomePage/components/WebpropertiesTable/Tutorials.tsx

```diff
@@ -0,0 +1,38 @@
+import { pageLinks } from '@app/links';
+import { Card, CardBody, CardHeader, SplitItem } from '@patternfly/react-core';
+
+const tutorialsData = [
+  {
+    title: 'SPAship Manager & CLI Guide',
+    link: 'https://drive.google.com/file/d/150OyktZdmqMXKwNS1mDkqIo7ZvgyCCGp/view'
+  },
+  {
+    title: 'On-Boarding to SPAship Cloud Native Version',
+    link: 'https://source.redhat.com/groups/public/spaship/blog_article/onboarding_to_spaship_cloud_native_version'
+  }
```

Comment:
@nikhitamore can we read it from the env variables

Reply:
done